apps/doji-backend/src/expert/expert.controller.ts

```diff
@@ -20,4 +20,12 @@ export class ExpertAppController {
     await this.expertAppService.applicationRequest(dto, user)
     return
   }
+  @Get('get-applications/')
+  async getAllApplicationList() {
+    return await this.expertAppService.getAllApplicationList()
+  }
+  @Get('get-applications/:keyword')
```

Comment:
keyword นี่หมายถึงไรนะ? expertUsername

Reply:
keyword คือ keyword ที่ใส่ใน search bar ของหน้า us7-1

Reply:
ใส่เป็น Query น่าจะดีกว่านะ (โดยทั่วไป filter มักจะมาเป็น query params เพราะมันใส่ก็ได้ ไม่ใส่ก็ได้)  e.g. `/applications?keyword=something` ดังนั้นเอาไปรวมกับ endpoint ข้างบนได้

แล้วก็ในกรณีนี้ service จะหน้าตาประมาณ `this.expertAppService.getAllApplicationList(filterOptions)` โดย 

```tsx
interface FilterOptions {
  keyword?: string
}
```